Mlem/Models/Trackers/Post Tracker.swift

```diff
@@ -25,53 +25,83 @@ class PostTracker: ObservableObject {
     func loadNextPage(account: SavedAccount, communityId: Int?, sort: SortingOptions?, type: FeedType) async throws {
         defer { isLoading = false }
         isLoading = true
-        
+
         let request = GetPostsRequest(
             account: account,
             communityId: communityId,
             page: page,
             sort: sort,
-            type: type
+            type: type,
+            limit: 50
         )
-        
+
         let response = try await APIClient().perform(request: request)
-        
+
         guard !response.posts.isEmpty else {
             return
         }
-        
+        Task(priority: .background) {
+            preloadImages(response.posts)
+        }
+
         add(response.posts)
         page += 1
     }
+
+    func preloadImages(_ newPosts: [APIPostView]) {
+        URLSession.shared.configuration.urlCache = AppConstants.urlCache
```

Comment:
This doesn't need changed in this PR, but once this is in it'd be an idea to move the pre-loading responsibility into a separate object, to me it feels we're muddying the responsibility of the post tracker somewhat with this, I think we should aim to keep the tracker solely focussed on handling posts, and have something else deal with image and other types of caching behaviours going forward